_data/simple-icons.json

```diff
@@ -15298,8 +15298,9 @@
         },
         {
             "title": "Tencent QQ",
-            "hex": "EB1923",
-            "source": "https://en.wikipedia.org/wiki/File:Tencent_QQ.svg"
+            "hex": "1EBAFC",
+            "source": "https://qq.design/brand/BrandDesign/Logo",
```

Comment:
Where are you getting a downloadable logo from? I'm only seeing PNG images.

Reply:
I just modified the previous one.

Reply:
Ah. Well my point stands that the link listed isn't really a source then.

Reply:
OK. Then let's use the `https://en.wikipedia.org/wiki/File:Tencent_QQ.svg` instead.

Updated.